src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,336 @@
+            [initialValue]="'default'"
+          >
+          </sky-docs-demo-control-panel-radio-group>
+        </sky-docs-demo-control-panel-section>
+      </sky-docs-demo-control-panel>
+
+      <sky-dropdown
+        [buttonType]="demoSettings.button"
+        [buttonStyle]="demoSettings.style"
+        >
+        <sky-dropdown-button>
+          Edit
+        </sky-dropdown-button>
+        <sky-dropdown-menu>
+          <sky-dropdown-item *ngFor="let item of items">
+            <button
+              type="button"
+              [attr.disabled]="item.disabled ? '' : null"
+              (click)="actionClicked(item.name)"
+            >
+              {{ item.name }}
+            </button>
+          </sky-dropdown-item>
+        </sky-dropdown-menu>
+      </sky-dropdown>
+    </sky-docs-demo>
+  </sky-docs-code-examples>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use dropdowns when users require access to multiple, related actions or when users require access to multiple, unrelated actions and you do not have room for multiple buttons.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-toolbar.png"
+          caption="use dropdowns to let users access multiple actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use context menu dropdowns for actions that affect individual items in a list.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-context-menu.png"
+          caption="use context menu dropdowns with items in a list."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use icon-only dropdowns when you do not have room for text labels. For example, use icon-only dropdowns in list toolbars when screen widths are narrow.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use icon-only dropdowns when there is not space to include a label."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use dropdowns as form inputs. Use HTML <sky-code>&lt;select&gt;</sky-code> fields instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
```

Comment:
I think this is the wrong image for this guideline.

Reply:
In the docs repo, the image here is `~/assets/img/guidelines/dropdown/usage-dont-input.png`. I think we need to copy that image over and update this `imageSource` property.